scripts/qa_crowbarsetup.sh

```diff
@@ -31,7 +31,7 @@ wanthyperv=
 
 export cloudfqdn=${cloudfqdn:-$cloud.cloud.suse.de}
 export nodenumber=${nodenumber:-2}
-export tempestoptions=${tempestoptions:--t -s}
+export tempestoptions=${tempestoptions:--subunit smoke}
```

Comment:
Also note that for the tests that we want to store in the rally server, we need to set `rally_server=backup.cloudadm.qa.suse.de`